core.lua

```diff
@@ -74,14 +204,25 @@ function CustomTooltips_DisplayTooltip(button, macroText)
 		local tooltipData = TOOLTIPS[tooltipName:upper()]
 		if tooltipData then
 			heading, body = tooltipData.heading, tooltipData.body
+		else
+			heading, body = SetNotFoundError(tooltipName)
 		end
 	else
-		heading, body = macroText:match("#tooltipdesc ([^\n]+)^([^\n]+)") -- #tooltipdesc heading text^body text
+		-- Do we have an inline tooltip definition?
+		if nil == string.find(macroText, "#tooltipdesc") then return end
```

Comment:
I prefer to invoke string functions as methods (`someStr:find(...)`) rather than through the `string` module (`string.find(someStr, ...)`).

I also prefer to use `not` (`if not someThing ...`) instead of explicit `nil` comparison (`if someThing == nil ...`).